docs/application/native/guides/alarm/alarms.md

```diff
@@ -32,8 +32,13 @@ To enable your application to use the alarm functionality:
 <privileges>
```

Comment:
Thanks for the update, but it looks like you forgot to add indentation for code blocks.

From line 31 to line 40, please add 3 spaces in front of each lines.
Also do the same to from line 43 to line 45.

Reply:
Done